### ISSUE TITLE
Use original source files

### ISSUE THREAD

**Issue by morganwillcock** [fonol/cl-community-spec#2]
The .info files are an output from the original documentation build. I might be wrong, but the original source files which created the .info files still look to be available: http://ftp.gnu.org/pub/gnu/gcl/old/gcl-info+texi.tgz

There are a lot of errors and warnings when building but it still (just about) builds:
```sh
wget http://ftp.gnu.org/pub/gnu/gcl/old/gcl-info+texi.tgz
tar -xzf gcl-info+texi.tgz
cd gcl-info
texi2any --html --error-limit=999999999 --force gcl.texi
xdg-open gcl/index.html
```

As a suggestion, it might be feasible to just update and fix the original texi files to work with current version of texinfo. From there you could export directly to HTML or to an intermediary format to pass to a different tool.

**Comment by fonol** [comment]
Hi, 
I actually tried on some texi files first with texi2any, but could not get it to compile. But with the files from your source, it actually works! I will look at the generated HTML in details, but judging from what I saw, it seems like a far better base to build on than the info files. Thanks so much, that will be very helpful.

**Comment by morganwillcock** [comment]
Possibly there are newer and already fixed source files. I haven't tried to build these, but from the commit history it looks like they were updated for newer versions of Texinfo:

https://git.savannah.gnu.org/cgit/gcl.git/tree/gcl/info

**Comment by fonol** [comment]
Thanks! I used the files from the first link you posted, and the result is already very pleasing. Much better than in my first version parsing the info files. I also already updated the live version at [cl-community-spec.github.io](https://cl-community-spec.github.io/pages/index.html). I will check if the script still runs the same on those and if there are any major differences, but for now, I think we are already at a pretty good point :)!
